README.md

```diff
@@ -161,6 +161,11 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What is event loop?
   * What is the difference between call stack and task queue?
 * Explain the differences on the usage of `foo` between `function foo() {}` and `var foo = function() {}`
+* What is inheritance?
+
+Inheritance (in programming) is a mechanism that allow sharing functionality (and code reuse) beetwen classes (in classical inheritance) or between objects (in prototypal inheritance - like in JS). Class that inherits (called subclass) gains acess to all shared behaviors and attributes of parent class (called superclass);
```

Comment:
w pierwszej linijce: allows
nie do końca jest to sendem problemu. W clasycznym dziedziczeniu też jest dziedziczeni po obiektach, z tą różnicą, że możesz mieć po klasach i obiektach. W prototypowym tylko jest linkowanie obiektów. Nie masz też problemu z tym, że będą się nadpisywały metody z dziedziczenia w jednej generacji po 2 klasach, jak to się zdarza w pythonie. Ale w JSie możesz też zaimplementować wiele rodzajów dziedziczenia
http://www.crockford.com/javascript/inheritance.html
http://stackoverflow.com/questions/19633762/classical-inheritance-vs-protoypal-inheritance-in-javascript